### PR TITLE
Improving how links can be written

### PR DESCRIPTION
**Pull request by mmattel**
This PR enhances the description in docs/best practices how links can be written/used.

### CODE REVIEW

File: docs/best-practices.md [hunk 1]
```diff
@@ -66,6 +66,18 @@ Please see [Troubleshooting Complex URLs](https://asciidoctor.org/docs/user-manu
 This is an example of an URL containing problematic characters which needs special treatment:
 `https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)`
 
+**Strongly** in favour using following method which greatly improves readbility during authoring the document.
```
Comment (settermjd) [inline docs/best-practices.md]: I'd change this to "Strongly in favour of using the following method which greatly improves readability while authoring the document"

File: docs/best-practices.md [hunk 2]
```diff
@@ -66,6 +66,18 @@ Please see [Troubleshooting Complex URLs](https://asciidoctor.org/docs/user-manu
...
+
+```
+= The Page Header
+:internal-link-name: https://example.com/content/link_can_be_very_long
+
+Text {internal-link-name}[highlighted text] text.
+```
+
+It is important, that `:internal-link-name:` is directly placed below the page header.
```
Comment (settermjd) [inline docs/best-practices.md]: I don't believe a comma is required before "that".

File: docs/best-practices.md (hunk 1 above)
Comment (mmattel) [reply docs/best-practices.md]: With or without bold "Strongly" ?

**Comment by mmattel** [comment]
All changes requested are fixed

File: docs/best-practices.md [hunk 3]
```diff
@@ -66,6 +66,18 @@ Please see [Troubleshooting Complex URLs](https://asciidoctor.org/docs/user-manu
...
+**Strongly** in favour of using the following method which greatly improves readability while authoring the document.
+
+```
+= The Page Header
+:internal-link-name: https://example.com/content/link_can_be_very_long
+
+Text {internal-link-name}[highlighted text] text.
+```
+
+It is important that `:internal-link-name:` is directly placed below the page header.
```
Comment (settermjd) [inline docs/best-practices.md]: s/directly placed/placed directly/g

File: docs/best-practices.md [hunk 4]
```diff
@@ -66,6 +66,18 @@ Please see [Troubleshooting Complex URLs](https://asciidoctor.org/docs/user-manu
...
+More of these link directives can be added in new lines.
```
Comment (settermjd) [inline docs/best-practices.md]: I'd suggest changing this to: "Any number of these link directives can be added."

**Comment by mmattel** [comment]
Updated based on feedback

**Comment by mmattel** [comment]
Based on @settermjd comment in a different PR, I added a text for adding `-url` in the placeholders name referencing the web address.

File: docs/best-practices.md [hunk 5]
```diff
@@ -66,6 +66,19 @@ Please see [Troubleshooting Complex URLs](https://asciidoctor.org/docs/user-manu
...
+**Strongly** in favour of using the following method which greatly improves readability while authoring the document.
+
+```
+= The Page Header
+:internal-link-name-url: https://example.com/content/link_can_be_very_long
+
+Text {internal-link-name-url}[highlighted text] text.
+```
+
+It is important that `:internal-link-name:` is placed directly below the page header.
```
Comment (settermjd) [inline docs/best-practices.md]: 👍

**Pull request merged**